draft-birkholz-cose-tsa-tst-header-parameter.md

```diff
@@ -113,6 +113,8 @@ The obtained timestamp token is then added back as an unprotected header into th
 
 In this context, timestamp tokens are similar to a countersignature {{-countersign}} made by the TSA.
 
+The message imprint sent to the TSA ({{Section 2.4 of -TSA}}) MUST be the hash of the payload field of the COSE signed object.
```

Comment:
this is not the gist of Carl's comment.  He wants us to factor out this, which is not CTT-specific:

    In either case, to minimize dependencies, the hash algorithm SHOULD
    be the same as the algorithm used for signing the COSE message.  This
    may not be possible if the timestamp token has been obtained outside
    the processing context in which the COSE object is assembled.